### ISSUE TITLE
aria-readonly="true" is not allowed on an element with contenteditable

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#353]
Steps:
- go to https://w3c.github.io/html-aria/tests/contenteditable-readonly-test.html
- run accessibility checker
- the violation for "The WAI-ARIA role or attribute 'aria-readonly' is not valid for the element `<div>`" is good, but there should also be a specific violation for aria-readonly="true" conflicting with contenteditable, as per https://w3c.github.io/html-aria/#att-contenteditable

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [x] testcases written (Dev)
- [ ] rule developed  (Dev)
- [ ] message written (SME)
- [ ] help written (SME)

**Comment by aliunwala** [comment]
Triage:
Looks like a rule fix/addition to hit the spec

**Comment by tombrunet** [comment]
@carmacleod Any reason that this testcase has no role on the div? If the thing is interactive, I kind of expect a role is needed? Looking at the accessibility tree in Chrome, it doesn't assign a role either (almost surprised it doesn't make it a textbox).

Some thoughts:
* [ ] Seems like anything with `contenteditable` should be included in the isTabbable logic
* [ ] Rule 1: Violation for conflict between contenteditable and readonly
* [ ] Rule 2: Recommendation for contenteditable and aria-readonly=false (Redundant)

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/353?planning-poker) with ZenHub @aliunwala @drjoho @mbgower @philljenkins @shunguoy @tombrunet

**Comment by shunguoy** [comment]
close as fixed per #800

**Comment by eirckr** [comment]
Validated 
![Uploading Screen Shot 2022-05-03 at 11.28.06 AM.png…]()
